src/shared/components/common.tsx

```diff
@@ -40,11 +40,8 @@ export const Badge = ({ content, outline = false }) => (
 export const DonateDesc = () => (
   <p className="text-sm text-gray-300 mb-3">
     <T i18nKey="donate_desc">
-      #
-      <Link className="link" to="/donate">
-        #
-      </Link>
-      #
+      <br />
+      <br />
     </T>
```

Comment:
I dont know how this thing works, but it creates the right newlines like this. Would also be better for lemmy-ui, to avoid using raw html.

Reply:
That's fine. The # are placeholders for the trans component: https://react.i18next.com/latest/trans-component#using-with-react-components

The help you insert html inside paragraphs / strings, IE in the `<1>stuff</1>` in the i18n `.json` .